.github/CONTRIBUTING.md

```diff
@@ -50,9 +50,6 @@ datum
 		var
 			varname1 = 1
 			varname2
-			static
```

Comment:
I know this is just a text file, but I don't want us to encourage doing something that's not allowed.